django/db/migrations/operations/utils.py

```diff
@@ -7,3 +7,23 @@ def is_referenced_by_foreign_key(state, model_name_lower, field, field_name):
                 if (f.to_fields[0] is None and field.primary_key) or field_name in f.to_fields:
                     return True
     return False
+
+
+def verbose_describe(operation, backwards):
+    if hasattr(operation, "code"):
+        action = operation.code.__doc__ if backwards is not True else operation.reverse_code.__doc__
+        prefix = ""
+    elif hasattr(operation, "sql"):
+        action = operation.sql if backwards is not True else operation.reverse_sql
+        prefix = ""
+    else:
+        action = "No further Detail"
```

Comment:
I'm not sure about the utility of `--> No further Detail`. In this case it might be better to just leave it off. 

(Maybe it's useful. Thoughts?)

Reply:
Initially, I was thinking about each action saying at least something to preserve the cadence of the output, but I agree it probably is completely superfluous.  I'll remove it.